test/model_level_tests/log2csv.py

```diff
@@ -0,0 +1,95 @@
+#==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# =============================================================================
+import argparse
+import os
+
```

Comment:
So the logs (calc_out.txt if i remember it right) are parsed by a file here: https://github.com/tensorflow/ngraph-bridge/blob/master/test/model_level_tests/models/ngraph-models/test1/custom_log_parser.py

This looks like a stand alone file that parses the "pass/fail info and throughput/latency info". Is the output file to be used for a dashboard?

In case we want to make the throughput/latency checks automated, we should add them in the custom parser for that test.

Reply:
This is meant for extracting the numbers and print them in a CSV - which cold be used for plotting graphs. Yes - we should add the validation login to the other file you created.